plugin/core/workspace.py

```diff
@@ -25,6 +31,9 @@ def includes_path(self, file_path: str) -> bool:
         else:
             return True
 
+    def __contains__(self, view: sublime.View) -> bool:
+        return self.includes_path(view.file_name() or '')
```

Comment:
`os.path.realpath()` with an empty string will return the working directory. Do we really want to follow that path or just return `False` early then?

Reply:
https://github.com/sublimelsp/LSP/pull/1069/commits/b20b02f4f2143aef978a3b7a99901291ec7cbf42